api-docs/leads.md

```diff
@@ -85,153 +85,76 @@ If there is no lead with the provided id, you will get a 404 HTTP response, with
+
 ### Response
 
 ```json
-[
-  {
-    "id": 1,
-    "dateOfSignUp": "2019-09-17",
-    "leadStatus": "active",
-    "contactStage": {
-      "first": "2019-05-06T06:00:00.000Z",
-      "second": null,
-      "third": null
-    },
-    "inactivityReason": null,
-    "eventSource": {
-      "eventId": 1,
-      "eventName": "Health Fair",
-      "eventLocation": "St Marks Hospital"
-    },
-    "firstName": "Harry",
-    "lastName": "Potter",
-    "fullName": "Harry Potter",
-    "phone": "5555555555",
-    "smsConsent": false,
-    "zip": "84115",
-    "age": 39,
-    "gender": "male",
-    "leadServices": [
-      {
-        "id": 1,
-        "serviceName": "Citizenship"
-      }
-    ],
-    "clientId": 1,
-    "isDeleted": false,
-    "clientId": 1,
-    "createdBy": {
-      "userId": 1,
-      "firstName": "Joel",
-      "lastName": "Denning",
-      "fullName": "Joel Denning",
-      "timestamp": "2019-05-06T06:00:00.000Z"
-    },
-    "lastUpdatedBy": {
-      "userId": 1,
-      "firstName": "Joel",
-      "lastName": "Denning",
-      "fullName": "Joel Denning",
-      "timestamp": "2019-05-06T06:00:00.000Z"
-    }
-  },
-  {
-    "id": 2,
-    "dateOfSignUp": "2019-09-17",
-    "leadStatus": "active",
-    "contactStage": {
-      "first": "2019-05-06T06:00:00.000Z",
-      "second": null,
-      "third": null
-    },
-    "inactivityReason": null,
-    "eventSource": {
-      "eventId": 1,
-      "eventName": "Health Fair",
-      "eventLocation": "St Marks Hospital"
-    },
-    "firstName": "Hermione",
-    "lastName": "Granger",
-    "fullName": "Hermione Granger",
-    "phone": "5555555555",
-    "smsConsent": true,
-    "zip": "84115",
-    "age": 40,
-    "gender": "female",
-    "leadServices": [
-      {
-        "id": 1,
-        "serviceName": "Citizenship"
-      }
-    ],
-    "clientId": 1,
-    "isDeleted": false,
-    "createdBy": {
-      "userId": 1,
-      "firstName": "Joel",
-      "lastName": "Denning",
-      "fullName": "Joel Denning",
-      "timestamp": "2019-05-06T06:00:00.000Z"
-    },
-    "lastUpdatedBy": {
-      "userId": 1,
-      "firstName": "Joel",
-      "lastName": "Denning",
-      "fullName": "Joel Denning",
-      "timestamp": "2019-05-06T06:00:00.000Z"
-    }
-  },
-  {
-    "id": 3,
-    "dateOfSignUp": "2019-09-17",
-    "leadStatus": "active",
-    "contactStage": {
-      "first": "2019-05-06T06:00:00.000Z",
-      "second": null,
-      "third": null
-    },
-    "inactivityReason": null,
-    "eventSource": {
-      "eventId": 1,
-      "eventName": "Health Fair",
-      "eventLocation": "St Marks Hospital"
-    },
-    "firstName": "Ron",
-    "lastName": "Weasley",
-    "fullName": "Ron Weasley",
-    "phone": "5555555555",
-    "smsConsent": true,
-    "zip": "84115",
-    "age": 39,
-    "gender": "male",
-    "leadServices": [
-      {
-        "id": 1,
-        "serviceName": "Citizenship"
+{
+  [
```

Comment:
This is invalid json (note the red highlight). Might be good to put future api doc json changes into an online json validador such as https://jsonlint.com/ before pushing the code, since there have been several changes recently where you’ve pushed invalid json.

The problem in the json is that you can’t put an array inside of an object without naming the array as a key in the object.